PATCH-P0-FIX-UPSTREAM: Incompatible types in ssl

Sometimes argument of this function can be const and function doesn't change
the variable so there is no reason not to make that argument const;

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

Index: vio/viosslfactories.c
===================================================================
--- vio/viosslfactories.c.orig
+++ vio/viosslfactories.c
@@ -198,7 +198,7 @@ static void check_ssl_init()
 static struct st_VioSSLFd *
 new_VioSSLFd(const char *key_file, const char *cert_file,
              const char *ca_file, const char *ca_path,
-             const char *cipher, SSL_METHOD *method, 
+             const char *cipher, const SSL_METHOD *method,
              enum enum_ssl_init_error* error)
 {
   DH *dh;
